corehq/util/zip_utils.py

```diff
@@ -66,3 +73,74 @@ def get(self, request, *args, **kwargs):
             response['Content-Length'] = os.path.getsize(fpath)
             set_file_download(response, self.zip_name)
             return response
+
+
+def iter_files_async(include_multimedia_files, include_index_files, app):
+    file_iterator = lambda: iter([])
+    errors = []
+    if include_multimedia_files:
+        from corehq.apps.hqmedia.views import iter_media_files
+        app.remove_unused_mappings()
+        file_iterator, errors = iter_media_files(app.get_media_objects())
+    if include_index_files:
+        from corehq.apps.app_manager.views import iter_index_files
+        if app.is_remote_app():
+            file_iterator, errors = iter_index_files(app)
+        else:
+            index_files, index_file_errors = iter_index_files(app)
+            if index_file_errors:
+                errors.append(index_file_errors)
+            file_iterator = itertools.chain(file_iterator, index_files)
```

Comment:
I think the `is_remote` check here is in the wrong place. The logic should be: only include multimedia if app is not remote